scripts/add_nodes_to_haamek_davar.py

```diff
@@ -0,0 +1,30 @@
+__author__ = 'stevenkaplan'
+from sefaria.helper.schema import *
+torah_titles = library.get_indexes_in_category("Torah")
+he_torah_titles = map(lambda x: library.get_index(x).get_title("he"), torah_titles)
+haamek_titles = map(lambda x: "Haamek Davar on {}".format(x), torah_titles)
+he_haamek_intros = map(lambda x: u"פתיחה לספר {}".format(x), he_torah_titles)
+en_haamek_intros = map(lambda x: "Introduction to {}".format(x), torah_titles)
+
+for title in haamek_titles:
+    print title
+    index = library.get_index(title)
+    convert_simple_index_to_complex(index)
+
+for i in range(5):
```

Comment:
The idiomatic way to do this is with [zip](https://docs.python.org/2/library/functions.html#zip):

`for title, hintro, eintro in zip(haamek_titles, en_haamek_intros, he_hammek_intros):`